docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
     environment:
       PORT: 8222
       MALLOC_ARENA_MAX: 2
-    command: ["-enable-url-source"]
+    command: ["-enable-url-source -forward-headers User-Agent"]
```

Comment:
This new option will need to be ported up to the infrastructure repository as well, if it is merged.